src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 10 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected 1, but got 4.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 43
43:             $result.Count          | Should -Be 1
```
Test-Connection.Ping.Count parameter
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 63
63:             $result1.Replies.Count | Should -Be 1
```
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Cannot index into a null array.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 91
91:             $result.Replies[0].Address              | Should -BeExactly $realAddress

```
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Cannot index into a null array.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 106
106:             $result1.Replies[0].Address              | Should -BeExactly $realAddress

```
Test-Connection.Ping.BufferSize works
```powershell
Cannot index into a null array.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 169
169:                 $result.Replies[0].Buffer.Count | Should -Be 2

```